lib/core/src/lib/pipes/adf-datetime.pipe.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { addMinutes, fromUnixTime, parse } from 'date-fns';
+
+@Pipe({ name: 'adfDateTime' })
+export class ADFDateTimePipe implements PipeTransform {
```

Comment:
Do we really need prefix class name with ADF? We have not prefixed other pipes like that, for example:
<img width="643" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/dc6dd7b2-c1f2-4a85-9a2a-8547865875e5">

adf as prefix in name (line 21st) should be enough as it already shows that it comes from ADF. Also when you imports that class then you can see that it comes from ADF.

Reply:
renamed class to DateTimePipe